js/wp-seo-replacevar-plugin-302.js

```diff
@@ -42,11 +53,68 @@
 			data = this.parentReplace( data );
 			data = this.doubleSepReplace( data );
 			data = this.excerptReplace( data );
+			data = this.categoryReplace( data );
+			data = this.customTaxonomyReplace( data );
+			data = this.tagReplace( data );
+
+			// Taxonomy replacements
+			if ( jQuery( 'input[name=tag_ID]').val() !== undefined ) {
+				data = this.taxonomyDescriptionReplace( data );
+			}
 		}
 		return data;
 	};
 
 	/**
+	 * Binding events for each taxonomy metabox element.
+	 *
+	 * @param {int} index
+	 * @param {Object} taxonomyMetaBox
+	 */
+	YoastReplaceVarPlugin.prototype.setTaxonomyEvents = function( index, taxonomyMetaBox ) {
+		// Set the events.
+		taxonomyMetaBox = jQuery( taxonomyMetaBox );
+
+		taxonomyMetaBox.on( 'wpListAddEnd', '.categorychecklist', function() {
+			this.fetchTaxonomies( taxonomyMetaBox );
+		}.bind( this ) );
+		taxonomyMetaBox.on( 'change', 'input[type=checkbox]', function() {
+			this.fetchTaxonomies( taxonomyMetaBox );
+		}.bind( this ) );
+
+		// We always wanted it to be fetched
+		this.fetchTaxonomies( taxonomyMetaBox );
+	};
+
+	/**
+	 * Adding tag events to the dom, to fetch added/removed tags.
+	 */
+	YoastReplaceVarPlugin.prototype.addTagEvents = function() {
+		if ( typeof MutationObserver === 'undefined' ) {
+			jQuery( '#post_tag' ).on( 'DOMSubtreeModified propertychange', '.tagchecklist', function() {
+				this.declareReloaded();
+			}.bind( this ) );
+		}
+		else {
+			// Select the target node
+			var target = document.querySelector('.tagchecklist');
```

Comment:
This can be done with jQuery.

To get the HTML object from the jQuery object you can use [jQuery's get function](http://api.jquery.com/get/).